script/addoperators-testnet.sh

```diff
@@ -1,5 +1,8 @@
 #!/bin/sh
 
-source .env
+# Fetch secrets from aws
+SECRET_JSON=$(aws secretsmanager get-secret-value --secret-id <SecretID> --query <SecretString> --output text)
```

Comment:
Will `<SecretID>` and `<SecretString>` be replaced with the values for testnet/mainnet once those secrets are defined in AWS?

Reply:
the code will be refactored to determine the environment, for the secrets ID and Secrets string of testnet/mainnet. 
could be something like

```
if [ "$1" == "testnet" ]; then
    SECRET_ID="your-testnet-secret-id"
elif [ "$1" == "mainnet" ]; then
    SECRET_ID="your-mainnet-secret-id"
else
    echo "Please specify environment: testnet or mainnet"
    exit 1
fi

SECRETS_JSON=$(aws secretsmanager get-secret-value --secret-id $SECRET_ID --query SecretString --output text)

DEPLOY_KEY=$(echo $SECRETS_JSON | jq -r .DEPLOY_KEY)
```
where we'll call 
```
./deploy.s.all.sh testnet  
./deploy.s.all.sh mainnet 
```